pkgs/top-level/all-packages.nix

```diff
@@ -25798,6 +25798,8 @@ in
 
   sentencepiece = callPackage ../development/libraries/sentencepiece {};
 
+  xp-pen-g430 = callPackage ../misc/drivers/xp-pen-g430 {};
```

Comment:
Should use `libsForQt5.callPackage`
https://nixos.org/nixpkgs/manual/#sec-language-qt